src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -495,15 +495,15 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 loop = true;
                 continue;
             }
-            result_wr = write(connFd, "\n", 2);
+            result_wr = write(connFd, "\r\n", 2);
 
             if (result_wr < 0) {
                 frontend_logger.log("Error writing to socket", "error");
                 loop = true;
                 continue;
             }
 
-            char type[20];
+            char type[21];
```

Comment:
Shall we move this literal 21 to a constant?

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/171/commits/389fdff2e5136ce9088e6d64a67cb4ca58dafe3d

Reply:
Moved the value 20 to `JasmineGraphFrontEndProtocol.h` as a constant `FRONTEND_GRAPH_TYPE_LENGTH`